include/glow/Graph/Graph.h

```diff
@@ -298,6 +298,19 @@ class Function final : public Named {
   LogNode *createLog(llvm::StringRef name, NodeValue input,
                      TypeRef outTy = nullptr);
 
+  /// Create a series of nodes with \p name that implements an element-wise
+  /// logit transform. For each element of the \p input x, this is
+  /// defined as:
+  ///
+  /// y = log(x / (1 - x))
+  ///
+  /// where the \p input is clamped in (\p eps, 1 - \p eps), and
+  /// the transform parameter \p eps is a small positive epsilon value.
```

Comment:
Suggestion: `small positive epsilon value` --> `small positive value`

Maybe it's a good idea to explain that `eps` is needed to avoid taking the logarithm of too small a value.

Reply:
Thanks! Added a more refined explanation!